src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -58,8 +58,8 @@ instance Default WalletStorage where
     def =
         WalletStorage
         {
-          _wsWalletMetas = mempty
-        , _wsProfile = mzero
+          _wsWalletMetas  = mempty
+        , _wsProfile      = Just def -- FIXME: Use CProfile, not Maybe CProfile
```

Comment:
I checked @akegalj. I'll delete this branch and create it from `0.4-rc` base so I don't pull changes from `master`.